<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSPracticeClass</class>
 <widget class="QMainWindow" name="RSPracticeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSPractice</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_input_mss">
          <property name="text">
           <string>输入（多光谱）：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_mss"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_mss">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_input_pan">
          <property name="text">
           <string>输入（全色）：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pan"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_pan">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_text">
        <property name="text">
         <string>选择融合方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_pca">
          <property name="text">
           <string>PCA（主成分分析）    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_pca">
          <property name="text">
           <string>输出路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pca"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_pca">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_gs">
          <property name="text">
           <string>GS（Gram-Schmidt）   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_gs">
          <property name="text">
           <string>输出路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_gs"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gs">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_ihs">
          <property name="text">
           <string>IHS                  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ihs">
          <property name="text">
           <string>输出路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ihs"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ihs">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_process">
        <property name="text">
         <string>处理</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RSPractice.qrc"/>
 </resources>
 <connections/>
</ui>
